--- go/src/runtime/cgo/cgo.go	2019-12-05 07:53:55.000000000 +0900
+++ patches/go/src/runtime/cgo/cgo.go	2020-01-03 17:40:41.000000000 +0900
@@ -12,8 +12,10 @@
 /*
 
 #cgo darwin,!arm,!arm64 LDFLAGS: -lpthread
-#cgo darwin,arm LDFLAGS: -framework CoreFoundation
-#cgo darwin,arm64 LDFLAGS: -framework CoreFoundation
+#cgo darwin,arm LDFLAGS: -framework CoreFoundation -isysroot /usr/share/SDKs/iPhoneOS.sdk -F/System/Library/Frameworks -miphoneos-version-min=7.0 -arch armv7 -L/usr/lib -L/usr/local/lib
+#cgo darwin,arm CFLAGS: -framework CoreFoundation -isysroot /usr/share/SDKs/iPhoneOS.sdk -F/System/Library/Frameworks -miphoneos-version-min=7.0 -arch armv7 -I/usr/include -I/usr/local/include -Wno-unused-command-line-argument
+#cgo darwin,arm64 LDFLAGS: -framework CoreFoundation -isysroot /usr/share/SDKs/iPhoneOS.sdk -F/System/Library/Frameworks -miphoneos-version-min=7.0 -arch arm64 -L/usr/lib -L/usr/local/lib
+#cgo darwin,arm64 CFLAGS: -framework CoreFoundation -isysroot /usr/share/SDKs/iPhoneOS.sdk -F/System/Library/Frameworks -miphoneos-version-min=7.0 -arch arm64 -I/usr/include -I/usr/local/include -Wno-unused-command-line-argument
 #cgo dragonfly LDFLAGS: -lpthread
 #cgo freebsd LDFLAGS: -lpthread
 #cgo android LDFLAGS: -llog
