--- luarocks-3.2.1/src/luarocks/core/cfg.lua	1567711414.000000000
+++ patches/luarocks-3.2.1/src/luarocks/core/cfg.lua	1577609694.786006000
@@ -439,6 +439,23 @@
       defaults.variables.CC = "env MACOSX_DEPLOYMENT_TARGET=10."..version.." gcc"
       defaults.variables.LD = "env MACOSX_DEPLOYMENT_TARGET=10."..version.." gcc"
       defaults.web_browser = "open"
+      if target_cpu == "aarch64" or target_cpu == "arm" then
+         defaults.variables.CC = "clang"
+         defaults.variables.LD = "clang"
+         defaults.variables.AR = "llvm-ar"
+         defaults.variables.RANLIB = "llvm-ranlib"
+         defaults.web_browser = "uiopen"
+         local arch = target_cpu
+         if arch == "aarch64" then
+            arch = "arm64"
+         elseif arch == "arm" then
+            arch = "armv7"
+         end
+         local FLAGS = " --sysroot=/usr/share/SDKs/iPhoneOS.sdk -I/usr/include -I/usr/local/include -L/usr/lib -L/usr/local/lib -F/System/Library/Frameworks -framework CoreFoundation -miphoneos-version-min=7.0 -arch "..arch
+         defaults.variables.CFLAGS = defaults.variables.CFLAGS..FLAGS
+         defaults.variables.LIBFLAG = defaults.variables.LIBFLAG..FLAGS.." -Wno-unused-command-line-argument"
+         defaults.gcc_rpath = true
+      end
    end
 
    if platforms.linux then
