--- go/src/cmd/go/internal/work/buildid.go	2019-12-05 07:53:55.000000000 +0900
+++ patches/go/src/cmd/go/internal/work/buildid.go	2020-01-03 16:55:12.000000000 +0900
@@ -678,6 +678,15 @@
 		if err := w.Close(); err != nil {
 			return err
 		}
+
+	// Khronos31: Rehash/sign binary (again) because it has changed.
+	// Hacky workaround for iOS codesigning.
+		if strings.Contains(target, "a.out") {
+			cmd := exec.Command("ldid", "-S/usr/share/SDKs/x.ent", target) // --> target string
+			if err := cmd.Run(); err != nil && (cfg.BuildX || cfg.BuildN) { // Just fail peacefully, the user can just sign this themself.
+				b.Showcmd("", "ldid -S %s # internal", target)
+			}
+		}
 	}
 
 	// Cache package builds, but not binaries (link steps).
