--- go/src/plugin/plugin_dlopen.go	1584631370.000000000
+++ patches/go/src/plugin/plugin_dlopen.go	1584686728.501221000
@@ -8,6 +8,10 @@
 
 /*
 #cgo linux LDFLAGS: -ldl
+#cgo darwin,arm LDFLAGS: -framework CoreFoundation -isysroot /usr/share/SDKs/iPhoneOS.sdk -miphoneos-version-min=7.0 -arch armv7
+#cgo darwin,arm CFLAGS: -framework CoreFoundation -isysroot /usr/share/SDKs/iPhoneOS.sdk -miphoneos-version-min=7.0 -arch armv7
+#cgo darwin,arm64 LDFLAGS: -framework CoreFoundation -isysroot /usr/share/SDKs/iPhoneOS.sdk -miphoneos-version-min=7.0 -arch arm64
+#cgo darwin,arm64 CFLAGS: -framework CoreFoundation -isysroot /usr/share/SDKs/iPhoneOS.sdk -miphoneos-version-min=7.0 -arch arm64
 #include <dlfcn.h>
 #include <limits.h>
 #include <stdlib.h>
