--- luarocks-3.2.1/src/luarocks/core/cfg.lua	1567711414.000000000
+++ patches/luarocks-3.2.1/src/luarocks/core/cfg.lua	1578242026.000000000
@@ -439,6 +439,17 @@
       defaults.variables.CC = "env MACOSX_DEPLOYMENT_TARGET=10."..version.." gcc"
       defaults.variables.LD = "env MACOSX_DEPLOYMENT_TARGET=10."..version.." gcc"
       defaults.web_browser = "open"
+      if target_cpu == "aarch64" or target_cpu == "arm" then
+         defaults.variables.CC = "clang"
+         defaults.variables.LD = "clang"
+         defaults.variables.AR = "llvm-ar"
+         defaults.variables.RANLIB = "llvm-ranlib"
+         defaults.web_browser = "uiopen"
+         local FLAGS = " -isysroot /usr/share/SDKs/iPhoneOS.sdk -miphoneos-version-min=7.0"
+         defaults.variables.LIBFLAG = defaults.variables.LIBFLAG..FLAGS.." -L/usr/local/lib -L/usr/lib"
+         defaults.variables.CFLAGS = defaults.variables.CFLAGS..FLAGS.." -I/usr/local/include -I/usr/include -Wno-unused-command-line-argument"
+         defaults.gcc_rpath = true
+      end
    end
 
    if platforms.linux then
